Add documentation for CLI interaction with registry having self-signed CA (#252)
vmware-tanzu/tanzu-cli: docs/full/README.md

   that signature verification is skipped for the repository. Users can choose to
   suppress this warning by setting the environment variable `TANZU_CLI_SUPPRESS_SKIP_SIGNATURE_VERIFICATION_WARNING`
   to `true`.

> ## Interacting with Central repository hosted on a registry with self-signed CA or with expired CA

Reviewer: ```suggestion
## Interacting with a central repository hosted on a registry with self-signed CA or with expired CA
```

Author: Updated. Thanks

> If user has configured central repository on custom registry (e.g. air-gaped environment) with self-signed CA or if the

Reviewer: ```suggestion
If a user has configured a central repository on a custom registry (e.g. air-gaped environment) with a self-signed CA or if the
```

Author: updated

registry CA

> certificate is expired, user can use the `tanzu config cert` command to configure the certificate for the registry host.

Reviewer: ```suggestion
certificate is expired, the user can execute the `tanzu config cert` family of commands to configure the certificate for the registry host.
```

Author: updated

```shell

    # If the registry host is self-signed add CA certificate for the registry

>     tanzu config cert add --host test.registry.com --ca-certificate path/to/ca/ert

Reviewer: ```suggestion
    tanzu config cert add --host test.registry.com --ca-certificate path/to/ca/cert
```

    # If the registry is self-signed and is serving on non-default port add CA certificate for the registry

>     tanzu config cert add --host test.registry.com:8443 --ca-certificate path/to/ca/ert

Reviewer: ```suggestion
    tanzu config cert add --host test.registry.com:8443 --ca-certificate path/to/ca/cert
```

Author: updated

    # If the registry is self-signed or CA cert is expired, add cert configuration for the registry host with
    # skip-cert-verify option
    tanzu config cert add --host test.registry.com  --skip-cert-verify true

    # Set to allow insecure (http) connection while interacting with host
    tanzu config cert add --host test.registry.com  --insecure true

```

> CLI uses the certificate configuration added for the registry host (using `tanzu config cert add` command ) while

Author: updated

interacting with the registry.

Users can update or delete the certificate configuration using the `tanzu config cert update`
and `tanzu config cert delete` commands.

> Also, users can list down the certificate configuration using `tanzu config cert list` command.

Reviewer: ```suggestion
Also, users can list the certificate configuration using the `tanzu config cert list` command.
```

Author: updated
